Services/data/devicemodels/scripts/prototype-01-state.js

```diff
@@ -90,15 +113,15 @@ function main(context, previousState, previousProperties) {
     // using the previous function state.
     restoreSimulation(previousState, previousProperties);
 
-    // 65 +/- 1%,  Min 35, Max 100
-    state.temperature = vary(65, 1, 35, 100);
+    // temperature +/- 1%,  Min 35, Max 100
+    state.temperature = vary(state.temperature, 1, 35, 100);
```

Comment:
Now that we support the SetTemperature method, we need to look at the state instead of resetting the temperature each time the simulation is run.